doc/code_style/mom.cpp

```diff
@@ -18,10 +18,10 @@
 #include "mom.h"
 
 // System includes follow, sorted alphabetically
-#include <iostream>
+#include <SDL/SDL.h>
 #include <cmath>
 #include <cstdio>
-#include <SDL/SDL.h>
+#include <iostream>
```

Comment:
include order should be strictly alphabetical

Reply:
SortIncludes (bool)
    If true, clang-format will sort #includes.
```
    false:                                 true:
    #include "b.h"                 vs.     #include "a.h"
    #include "a.h"                          #include "b.h"
```
This is sadly the only option here. I think clang-format treats `SDL/SDL.h` different because it's not part of the standard library and for that treated as external.

What about if the let clang-format do it this way, but as long as the grouping is preserved. It has also the advantage, that you see directly on top of this group which non-standard header was included. What do you think?

Reply:
yea that's probably fine, as long as the groups are preserved.